### ISSUE TITLE
CTD on final approach

### ISSUE THREAD

**Issue by JeremyGolf70** [flybywiresim/a32nx#5628]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-08-11T07:29:20+00:00",
  "ref": "master",
  "sha": "1c5335ec13ec6b7d29319db3371299da5e4fb880",
  "actor": "davidwalschots",
  "event_name": "manual"
}
```

### Describe the bug

Once the aircraft perform the final approach, MSFS crashes to desktop. Every time. Never landed on solid ground. 

### Expected behavior

Landed safely, no CTDs. 

### Steps to reproduce

1. Make a flightplan on SimBrief.
2. Set up everything including callsign, tailnumber, fuel and weight in the aircraft selection menu. 
3. Set up a departure bay. 
4. Start the flight. 
5. During the flight, I would hear the engine lost its sound in seconds and the aircraft froze  in the air too.
6. On approach to the airport, the aircraft freeze in a short time but more frequently. 
7. On around 500ft above the ground to touch down, it CTD. 
8. I have tried to land on VHHH, NZWN and NZQN.

**Comment by Fed16** [comment]
Same for me, i came across various ctds in the same condition except for the fifth step (i've not experienced engine sound loss during the flights). This happened in differents airports and with differents approaches types (ILS or RNAV).

**Comment by hotshotp** [comment]
I mean you don't hear the engine sound at cruise other than a faint rumble, which is true to life. Not sure about the CTD, I've not had that issue.

**Comment by ErickSharp** [comment]
This could be caused by a variety of reasons. One could be that your system is not able to put up with the amount of rendering it has to when you get closer to the ground. Are you experiencing extremely high RAM usage just before your CTD's?

**Comment by quinti123** [comment]
same here on 25L EDDF

**Comment by JeremyGolf70** [comment]
> This could be caused by a variety of reasons. One could be that your system is not able to put up with the amount of rendering it has to when you get closer to the ground. Are you experiencing extremely high RAM usage just before your CTD's?

I opened the Developer mode and found that the GPU suffers a lot. I have got 2 sets of 16GB RAM but the maximum usage was even less than 16GB. 
Also, I have changed an AI aircraft mod and it wouldn't CTD anymore, but stuck a lot. 

My computer is equipped with I7-10700 and 2060Super. 16GB*2=32GB RAM. 
![Microsoft Flight Simulator Screenshot 2021 08 15 - 11 34 53 24](https://user-images.githubusercontent.com/49427357/129467343-07d22c9a-5934-4e20-a914-5ee3b6e397f8.png)
![Microsoft Flight Simulator Screenshot 2021 08 15 - 12 13 30 33](https://user-images.githubusercontent.com/49427357/129467345-d43af2ba-62bb-4c85-a3f9-02cc9800ffcf.png)
![Microsoft Flight Simulator Screenshot 2021 08 15 - 11 59 19 37](https://user-images.githubusercontent.com/49427357/129467346-6f06111c-0946-4848-b709-d9f287ede236.png)

**Comment by JeremyGolf70** [comment]
> I mean you don't hear the engine sound at cruise other than a faint rumble, which is true to life. Not sure about the CTD, I've not had that issue.

Sorry about my terrible English skills. 
Actually, I just have found that, the whole simulation stuck for a second or several second, and that is why the engine sound lost for several seconds. It stuck many times during the cruise and stuck even more and longer during approach.

**Comment by donchoo** [comment]
I wouldn't call mine a CTD, but with the last two dev versions (current 2019690), I'm getting random crashes (as in the sim says that my plane has crashed) during the initial climb to 1000ft at 15 deg AOA, I'm hesitant but it seems to occur when I'm taking off into a strong headwind i.e. 20kts+ still trying to replicate this issue.

**Comment by NeoEagle** [comment]
Thats so weird. The CTDs drive me crazy... Mostly i can land on hard surfaces but often it simply crashes. I did a complete reinstall as described by the FBW Team + reinstall of the sim, NO Traffic, NO Community contents but the FBW and I don't get rid of the problems. But there is one thing i changed recently: In my opinion the CTS's seem to happen more often after I began to load my aircraft by the MCDU builtin "loader" (simbrief importer). But I don't know if there is any relation. I recently recorded an approach while it crashed -> The last 10 maybe 20 seconds are the more interesting part, where it goes black, there it crashes. Am I landing too smooth? ^^ https://youtu.be/9g2ugiSyLec
